sparrow-py/pytests/golden/collect_test/test_collect_bool_trailing_window_1s.jsonl

```diff
@@ -1,6 +1,12 @@
 {"_time":"1996-12-19T16:39:57.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[true]}
-{"_time":"1996-12-19T16:39:58.000","_subsort":1,"_key_hash":2867199309159137213,"_key":"B","b":true,"collect_b":[true]}
-{"_time":"1996-12-19T16:39:59.000","_subsort":2,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[true]}
-{"_time":"1996-12-19T16:40:00.000","_subsort":3,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false]}
-{"_time":"1996-12-19T16:40:01.000","_subsort":4,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false,false]}
-{"_time":"1996-12-19T16:40:02.000","_subsort":5,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[false,true]}
+{"_time":"1996-12-19T16:39:58.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:39:59.000","_subsort":1,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:00.000","_subsort":1,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:00.000","_subsort":2,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false]}
+{"_time":"1996-12-19T16:40:01.000","_subsort":2,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:01.000","_subsort":3,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false]}
+{"_time":"1996-12-19T16:40:02.000","_subsort":3,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:02.000","_subsort":4,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[true]}
+{"_time":"1996-12-19T16:40:03.000","_subsort":4,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:04.000","_subsort":5,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
+{"_time":"1996-12-19T16:40:05.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[]}
```

Comment:
These are the results I expect. However, if you uncomment the 
```
       # return collect.filter(is_input)
```
and rerun this test, the results change unexpectedly. I presume it is some continuity behavior happening, but unsure. 

```
{"_time":"1996-12-19T16:39:57.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[true]}
{"_time":"1996-12-19T16:39:59.000","_subsort":1,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[true]}
{"_time":"1996-12-19T16:40:00.000","_subsort":2,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false]}
{"_time":"1996-12-19T16:40:01.000","_subsort":3,"_key_hash":12960666915911099378,"_key":"A","b":false,"collect_b":[false]}
{"_time":"1996-12-19T16:40:02.000","_subsort":4,"_key_hash":12960666915911099378,"_key":"A","b":true,"collect_b":[true]}
{"_time":"1996-12-19T16:40:04.000","_subsort":5,"_key_hash":12960666915911099378,"_key":"A","b":null,"collect_b":[true]}

```

Note that the second row also contains a `[true]`, as well as the last row. It may be storing the incorrect values in the latched spread, as we can tell that without the filter, we are producing an empty list (and not null). However, that leads to the conclusion that there may be a bug in the latched spread, which I'm hesitant to blame without more evidence.